### PR TITLE
remove RC language from what's new

### PR DESCRIPTION
**Pull request by mairaw**
Let's merge this once the release is live so we can confirm the URLs.

Contributes to #14216

### CODE REVIEW

File: docs/core/whats-new/dotnet-core-3-0.md [hunk 1]
```diff
@@ -6,49 +6,34 @@ dev_langs:
...
 
 The MSI installer for Windows has changed starting with .NET Core 3.0. The SDK installers will now upgrade SDK feature-band releases in place. Feature bands are defined in the *hundreds* groups in the *patch* section of the version number. For example, **3.0._101_** and **3.0._201_** are versions in two different feature bands while **3.0._101_** and **3.0._199_** are in the same feature band. And, when .NET Core SDK **3.0._101_** is installed, .NET Core SDK **3.0._100_** will be removed from the machine if it exists. When .NET Core SDK **3.0._200_** is installed on the same machine, .NET Core SDK **3.0._101_** won't be removed.
 
 For more information about versioning, see [Overview of how .NET Core is versioned](../versions/index.md).
 
-## C# 8.0 preview
+## C# 8.0
 
-.NET Core 3.0 supports C# 8 preview. For more information about C# 8.0 features, see [What's new in C# 8.0](../../csharp/whats-new/csharp-8.md).
+C# 8 is also part of this release, which includes nullable, async streams, and more patterns. For more information about C# 8.0 features, see [What's new in C# 8.0](../../csharp/whats-new/csharp-8.md).
```
Comment (pkulikov) [inline docs/core/whats-new/dotnet-core-3-0.md]:
```suggestion
C# 8.0 is also part of this release, which includes nullable, async streams, and more patterns. For more information about C# 8.0 features, see [What's new in C# 8.0](../../csharp/whats-new/csharp-8.md).
```

File: docs/core/whats-new/dotnet-core-3-0.md [hunk 2]
```diff
@@ -6,49 +6,34 @@ dev_langs:
...
-- [.NET Core 3.0 Preview 3 announcement](https://devblogs.microsoft.com/dotnet/announcing-net-core-3-preview-3/)
-- [.NET Core 3.0 Preview 2 announcement](https://devblogs.microsoft.com/dotnet/announcing-net-core-3-preview-2/)
-- [.NET Core 3.0 Preview 1 announcement](https://devblogs.microsoft.com/dotnet/announcing-net-core-3-preview-1-and-open-sourcing-windows-desktop-frameworks/)
-
-## Production supported preview
-
-.NET Core RC1 is considered production ready by Microsoft and is fully supported. Starting with preview 7, releases will focus on polishing .NET Core 3.0 instead of adding new features. For more information about what has changed in RC1, see the [RC1 announcement](https://devblogs.microsoft.com/dotnet/announcing-net-core-3-0-release-candidate-1/).
-
-If you're using a previous preview release, you must move to RC1 for continued "Go Live" support.
+.NET Core RC1 was considered production ready by Microsoft and was fully supported. If you're using a preview release, you must move to the RTM version for continued support.
```
Comment (rpetrusha) [inline docs/core/whats-new/dotnet-core-3-0.md]:
```suggestion
.NET Core RC1 is considered production ready by Microsoft and is fully supported. If you're using a preview release, you must move to the RTM version for continued support.
```

**Review by rpetrusha — APPROVED** [review]
I've left one suggestion, @mairaw.

File: docs/core/whats-new/dotnet-core-3-0.md (hunk 2 above)
Comment (Thraka) [reply docs/core/whats-new/dotnet-core-3-0.md]: In general it's been required that a customer move to the next preview/rc release to continue support. I'm assuming that it means the same for RTM. The change you're suggesting makes it sound like RC1 is still supported even though RTM is released.

**Review by Thraka — APPROVED** [review]

File: docs/core/whats-new/dotnet-core-3-0.md (hunk 2 above)
Comment (mairaw) [reply docs/core/whats-new/dotnet-core-3-0.md]: Indeed. The change was by design to make it clear they need to move to RTM.

**Pull request merged**